pom.xml

```diff
@@ -286,7 +286,7 @@
             <dependency>
                 <groupId>ch.qos.logback</groupId>
                 <artifactId>logback-classic</artifactId>
-                <version>1.4.7</version>
+                <version>1.3.7</version>
```

Comment:
Confirmed with @jeromevdl that the change as 1.4.x is not compatible with Java 8. 
I suggest changing it to latest 1.3.x version (1.3.8)

Reply:
done